frontend/src/concepts/pipelines/content/createRun/contentSections/RunTypeSectionScheduled.tsx

```diff
@@ -31,98 +15,14 @@ type RunTypeSectionScheduledProps = {
 const RunTypeSectionScheduled: React.FC<RunTypeSectionScheduledProps> = ({ data, onChange }) => (
   <Stack hasGutter>
     <StackItem>
-      <Text>
-        <b>Trigger type</b>
-      </Text>
+      <TriggerTypeField data={data} onChange={onChange} />
     </StackItem>
     <StackItem>
-      <Radio
-        name="run-type-option-trigger-type"
-        label="Periodic"
-        isChecked={data.triggerType === ScheduledType.PERIODIC}
-        id={ScheduledType.PERIODIC}
-        onChange={() =>
-          onChange({
-            ...data,
-            triggerType: ScheduledType.PERIODIC,
-            value: DEFAULT_PERIODIC_OPTION,
-          })
-        }
-        body={
-          data.triggerType === ScheduledType.PERIODIC && (
-            <>
-              <Text>
-                <b>Run every</b>
-              </Text>
-              <Split hasGutter>
-                <SplitItem>
-                  <NumberInputWrapper
-                    min={1}
-                    value={parseInt(data.value) || 1}
-                    onChange={(value) =>
-                      onChange({
-                        ...data,
-                        value: replaceNumericPartWithString(data.value, value),
-                      })
-                    }
-                  />
-                </SplitItem>
-                <SplitItem>
-                  <SimpleDropdownSelect
-                    options={Object.values(PeriodicOptions).map((v) => ({
-                      key: v,
-                      label: v,
-                    }))}
-                    value={data.value.replace(/\d+/, '')}
-                    onChange={(value) =>
-                      onChange({
-                        ...data,
-                        value: replaceNonNumericPartWithString(data.value, value),
-                      })
-                    }
-                  />
-                </SplitItem>
-              </Split>
-            </>
-          )
-        }
+      <MaxConcurrencyField
+        onChange={(maxConcurrency) => onChange({ ...data, maxConcurrency })}
```

Comment:
I am able to set the value to 0 which throws an error on submit. not by arrow but by typing

Reply:
same thing with setting large numbers over 10

Reply:
Thanks for catching that, fixed and added a test for it.